### ISSUE TITLE
merit feature : non-hexadecimal number found in fromhex()

### ISSUE THREAD

**Issue by tomitomas** [KaSroka/Toshiba-AC-control#11]
Hello Kamil, 

There's one more issue on `toshiba_ac/device.py` file for the function `load_supported_merit_features`, line 117

as example : 
```
import struct

# data recevied by the unit  
# ToshibaAcDeviceInfo(ac_id='xxx', ac_unique_id='xxxx', ac_name='Room', initial_ac_state='31421535316400101410fe0b00001002000000', firmware_version='3.2.01', merit_feature='ac00', ac_model_id='2'),
merit_feature_hexstring='ac00'

try:
    merit_byte, = struct.unpack('b', bytes.fromhex(merit_feature_hexstring[:2]))
except (TypeError, ValueError, struct.error):
    ac_model_id = '1'
        
floor, _, cdu_silent, pure_ion, fireplace, heating_8c, _, _ = struct.unpack('????????', bytes.fromhex('0' + '0'.join(f'{merit_byte:08b}')))
```

exception raised : 
``` 
ValueError: non-hexadecimal number found in fromhex() arg at position 1
```

**Comment by KaSroka** [comment]
Thanks for reporting this.

It will work as you have described.
Another possibility is to change from:
```python
merit_byte, = struct.unpack('b', bytes.fromhex(merit_feature_hexstring[:2]))
```
to
```python
merit_byte, = struct.unpack('B', bytes.fromhex(merit_feature_hexstring[:2]))
```
As this was my oversight - `b` means signed, `B` means unsigned.

This should be also changed in `fcu_state.py` for non temperature fields.

**Comment by ocampana** [comment]
Hello, I opened the bu in the home-assistant-toshiba . I tried replacing 'b' with 'B', as proposed by @KaSroka  and it works perfectly!

**Comment by ocampana** [comment]
I opened the pull request #15 . This fixes both the script and home assistant.

**Comment by KaSroka** [comment]
Fixed in 0.1.14
